src/components/Accordion/__snapshots__/Accordion.spec.jsx.snap

```diff
@@ -13,7 +13,7 @@ exports[`Accordion [common] example testing should match snapshot(s) for 1.basic
     className="lucid-Accordion-Item"
     isExpanded={false}
     onToggle={[Function]}>
-    Raymond \"Ray\" Stantz, Ph.D. is a fictional character from the Ghostbusters franchise. He appears in the films Ghostbusters, Ghostbusters II, Casper, and the animated television series The Real Ghostbusters. He was portrayed by Dan Aykroyd in both live action films, and voiced by Frank Welker in the animated series. He is a member of the Ghostbusters and one of the three doctors of parapsychology, along with Dr. Peter Venkman and Dr. Egon Spengler.
```

Comment:
looks like just escapes got removed from the output, maybe a minor bump of jest or something?